README.markdown

```diff
@@ -76,13 +76,24 @@ You will need:
    export GOOGLE_APP_ID="Google Project Id"
    ```
 
-   **Obtaining Google Credentials:** 
+   **Obtaining Google Credentials:**
+
        * Sign in to Google Developer Console
+
        * Create a project (Use this id as GOOGLE_APP_ID)
```

Comment:
Maybe you should walk me through this. I'm having trouble following the steps, but I've never used this before so maybe I'm missing something obvious.